hearth/sig/lib/hearth/interceptor_context.rbs

```diff
@@ -1,6 +1,6 @@
 module Hearth
   class InterceptorContext
-    def initialize: (input: Structure, request: Request, response: Response, output: Output[Structure], logger: Logger) -> void
+    def initialize: (input: Structure, request: Request, response: Response, output: Output[Structure], config: Configuration[instance]) -> void
```

Comment:
What is instance here?  Should this be based on the service and be a type parameter on InterceptorContext (And should Request, Response and Output use it?)